src/service/video/OWNERS.yaml

```diff
@@ -0,0 +1,2 @@
+ - alanorozco
```

Comment:
Please add the standard OWNERS file comment here (see any other file for examples)